### PR TITLE
Correct inaccuracies in Observability Pipelines docs

### PR DESCRIPTION
**Pull request by jhgilbert**
### What does this PR do? What is the motivation?
DOCS-7764

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/observability_pipelines/log_volume_control/sumo_logic_hosted_collector.md
[the remark this answers is not in the document]
Comment (jhgilbert) [reply content/en/observability_pipelines/log_volume_control/sumo_logic_hosted_collector.md:1]: We include a "nothing to do here" message if the step is in a set of tabs, but not if it's on a standalone page. 

In a set of tabs, we want to make sure the user understands all tech is supported, which is why we include a tab header even when nothing is required (so the user can read the tab headers at the top and grasp that all are supported). 

But in non-tab content, the user is more likely to see us as wasting their time by including an extra step that does not actually require anything of them, so better to omit it in that case.

**Review by cswatt — APPROVED** [review]
lgtm!

File: content/en/observability_pipelines/log_volume_control/sumo_logic_hosted_collector.md
[the remark this answers is not in the document]
Comment (Akula112233) [reply content/en/observability_pipelines/log_volume_control/sumo_logic_hosted_collector.md:1]: Got it, thank you!

**Review by Akula112233 — APPROVED** [review]
Looks good, thanks for the changes!

**Pull request merged**